src/js/edu-benefits/pages/schoolSelection.js

```diff
@@ -4,21 +4,38 @@ import * as educationProgram from '../definitions/educationProgram';
 import { uiSchema as uiSchemaDate } from '../../common/schemaform/definitions/date';
 import { civilianBenefitsLabel } from '../utils/helpers';
 
-const defaults = {
-  fields: [
-    'educationProgram',
-    'educationObjective',
-    'nonVaAssistance'
-  ],
-  required: [
-    // 'educationType',
-    // 'name'
-  ]
+
+// Exported like this so we can share the fields between the formConfig and the tests.
+export const schoolSelectionOptionsFor = {
```

Comment:
I think defaults being gone is an improvement at this point, but it bugs me that the configuration is in this file instead of somewhere in the folder for the specific form. It forces you to come here, in our common edu code, to see form specific details. Those details don't seem like they should be the concern of the common page code.